### PR TITLE
Added missing French strings

### PR DESCRIPTION
**Pull request by loicsans**
**Description**

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Jira link:

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Passes test case
- [ ] UI components visually reviewed on desktop view
- [ ] UI components visually reviewed on mobile view
- [ ] Other (please explain)

**Checklist:**

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The precommit and linting ran successfully
- [ ] I have added or updated language tags for text that's part of the UI
- [ ] I have added "MISSING" for all new language tags to languages I don't speak
- [ ] I have added [the GNU General Public License](https://lite-farm.atlassian.net/l/cp/BT0Dd7WW) to all new files

### CODE REVIEW

File: packages/webapp/public/locales/fr/translation.json
Comment (SayakaOno) [inline packages/webapp/public/locales/fr/translation.json:543]: Thank you for the PR Loïc!
It seems like line 510 -526 should be updated instead of adding these!

Comment (SayakaOno) [reply packages/webapp/public/locales/fr/translation.json]: Update: I addressed this!

**Comment by SayakaOno** [comment]
@loicsans I removed duplicated keys and fixed conflicts. Could you double check that the translations you added are still good?
@antsgar It would be great if you could merge this once Loïc approved!
Thank you!

**Comment by kathyavini** [comment]
@loicsans if we could also please get one more change added into this PR, we need CROP_SALE in `revenue.json` updated from its current, which just reads "sale" to something that specifies crops 🙏

**Review by antsgar — APPROVED** [review]

**Pull request merged**
